freeze/boot.py

```diff
@@ -68,8 +68,10 @@ def __enter__(self):
 
     def __exit__(self, cls, err, tb):
         if err:
-            error_handler(cls, err, tb, self._action)
-            sys.exit(1)
+            if not isinstance(err, SystemExit) or err.code:
+                error_handler(cls, err, tb, self._action)
+                if not isinstance(err, SystemExit):
+                    sys.exit(1)
```

Comment:
Thanks!  I don't think all exceptions have a `.code` though, so let's remove that? Or am I missing something?
```suggestion
            if not isinstance(err, SystemExit):
                error_handler(cls, err, tb, self._action)
                sys.exit(1)
```

Reply:
I guess you will surely already know the answer:
The expression on the right of the `or` will only be evaluated if the left part equals `False`, because of the short circuit evaluation.
Therefore `err.code` is only accessed if `err` is an instance of `SystemExit`.

Reply:
Ah yeah, I missed that it'd short-circuit. Good call to cover this case 👍